### PR TITLE
Fix/format returned tweets

### PR DESCRIPTION
**Pull request by jungeunyoon**
- passed manual tests

### CODE REVIEW

**Comment by jungeunyoon** [comment]
[Address Twitter API edge case where a single tweet is returned [2]](https://trello.com/c/A74fd8LU/4758-address-twitter-api-edge-case-where-a-single-tweet-is-returned-2)

File: package.json [hunk 1]
```diff
@@ -45,6 +45,7 @@
   },
   "dependencies": {
     "common-display-module": "git://github.com/Rise-Vision/common-display-module.git#v2.1.4",
+    "rewire": "^3.0.2",
```
Comment (jungeunyoon) [inline package.json]: Needed to unit test unexposed functions.
